File: FullStack_Developer/1_java/workspace/day05/src/day05/ClassTest.java
package day05;

public class ClassTest {
	public static void main(String[] args) {
//		Car mycar = new Car();
////		System.out.println(mycar);
//		mycar.model = "Ferrari";
//		mycar.color = "Red";
//		mycar.price = 70000;
//		System.out.println(mycar.model);
		
		Car mycar = new Car("Ferrari","Red",70000);
		System.out.println(mycar.model);
		Car momcar = new Car("K7", "White", 4000);
		System.out.println(momcar.model);
		Car dadcar = new Car("Genesis G80", "Black", 8000);
		
		mycar.engineStart();
		momcar.engineStart();
		dadcar.engineStart();
	}
}
class Car{
	//속성, 성질들은 변수로 선언
	String carnum;
	String model;
	String color;
	int price;
	
	Car(String model, String color, int price){
		this.model = model;
		this.color = color;
		this.price = price;
	}
	
	//기능, 행위들은 메소드로 선언
	void engineStart() {
		System.out.println(this.model+" 시동 켜기");
	}
	
	void engineStop() {
		//전역변수 model이 선언된 것이 없으므로 this.은 생략 가능
		System.out.println(model+" 시동 끄기");
	}
}

File: Basic_Language/2_Python/workspace/day06/ClassTest.py
#ClassTest.py
'''
mycar_brand = "Ferrari"
mycar_color = "Red"
mycar_price = 65000

def mycar_engineStart():
'''
class Car:
    brand = ''
    color = ''
    price = 0

    def engineStart(self):
        print(self.brand + " 시동 켜기")

    def engineStop(self):
        print(self.brand + " 시동 끄기")

mycar = Car()
mycar.brand = "Ferrari"
mycar.color = "Red"
mycar.price = 65000

momcar = Car()
momcar.brand = "K7"
momcar.color = "White"
momcar.price = 7000

print(mycar.brand)
mycar.engineStart()
momcar.engineStart()

print(Car.brand)




















